apps/question/views.py

```diff
@@ -36,8 +37,10 @@ def get(self, request, **kwargs):
         aa.question.id
```

Comment:
`answered_questions` could be extracted to an `ExamAttempt` property

Reply:
Oops, I missed this one. I'll fix it with next commit.